src/Particle.js

```diff
@@ -297,6 +304,23 @@ class Particle {
 		});
 	}
 
+	/**
+	 * Revoke all active access tokens
+	 * @param {Object} options            Options for this API call
+	 * @param {String} options.auth       Access Token
+	 * @param {Object} [options.headers]  Key/Value pairs like `{ 'X-FOO': 'foo', X-BAR: 'bar' }` to send as headers.
+	 * @param {Object} [options.context]  Request context
+	 * @returns {Promise} A promise
```

Comment:
going forward, i'd love it if we could specify what the promise resolves or rejects with - e.g

```
@returns {Promise<ApiDeleteTokenResponse|ApiError>} A promise for the api response or an api error
```

Reply:
Yep, that's a good idea. I don't think we have any clear types coming back from the promise (so they just keep resolving `Object` 🤷‍♂️ ).